lib/exec.js

```diff
@@ -10,7 +10,7 @@ module.exports = function exec(script) {
 
 	script.env = script.env || {};
 
-	var env = objectAssign(process.env, script.env);
+	var env = objectAssign({}, script.env, process.env);
```

Comment:
@just-boris This diff introduced a breaking change that reversed the logic for applying environment variables.

_Previously_ `script.env` values would override `process.env` values.

_After this change_ `process.env` values (e.g. loaded from `.env` file) override `script.env` values (variables defined in `betterScripts` env block).

/cc @benoror #56